packages/core/src/common/command.ts

```diff
@@ -118,6 +118,15 @@ export class CommandRegistry implements CommandService {
         return this.doRegisterCommand(command);
     }
 
+    /**
+     * Unregister command by it's identifier.
+     */
+    unregisterCommand(commandId: string) {
```

Comment:
please rollback, we use `Disposable` pattern, `registerCommand` return a disposable object which should be disposed to unregister

Reply:
done